webapp/Development.Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.9.10 as webapp
+
+WORKDIR /app
+
+RUN pip install --upgrade pip pipenv flask python-dotenv
+COPY ./Pipfile ./Pipfile.lock ./
+RUN pipenv install --system
+RUN rm .env -f
+# Now copy in our code, and run it
+COPY . .
+EXPOSE 8000
```

Comment:
Wouldn't do that, because runs only once at build. With the other solution it runs every time when starting environment and also you can trigger it to run separately.

Reply:
I am just wondering why you would need that functionality? If we do not need to build a new image the texts should also not have changed or do I overlook sth?

Reply:
So everytime texts change we need to build a new image, what wouldn't be necessary if we have it in docker compose. Do you have other considerations about what I am missing?